subcommand/lifecycle-sidecar/command.go

```diff
@@ -27,16 +26,15 @@ type Command struct {
 	flagSet           *flag.FlagSet
```

Comment:
Since we're updating things, it would be good to switch the `flagSyncPeriod` to a `flags.DurationValue` instead. The sync process has an [example](https://github.com/hashicorp/consul-k8s/blob/master/subcommand/sync-catalog/command.go#L45).

Reply:
Yeah, good call. I've updated it to use `time.Duration` type and parse it with `DurationVar` function instead. It made it easier to handle the default, plus I didn't see the need for a pointer here. Let me know what you think.